draft-ietf-quic-transport.md

```diff
@@ -470,7 +472,9 @@ this version, it contains:
   * 01: 1-RTT packet (packet number size = 1)
   * 02: 1-RTT packet (packet number size = 2)
   * 03: 1-RTT packet (packet number size = 4)
+  * 07: 1-RTT packet (packet number size = 4, with packet number echo)
```

Comment:
Can add two additional codepoints here:
```
* 05: 1-RTT packet (packet number size = 1, packet number echo size = 1)
* 06: 1-RTT packet (packet number size = 2, packet number echo size = 2)
```

(I remain skeptical of the tradeoff for varlen packet numbers, but if we're going to have them, we might as well have varlen echo as well.)